apps/studio/components/interfaces/Home/ProjectList/ProjectCard.tsx

```diff
@@ -25,7 +25,7 @@ const ProjectCard = ({
   const { name, ref: projectRef } = project
   const desc = `${project.cloud_provider} | ${project.region}`
 
-  const isBranchingEnabled = project.preview_branch_refs.length > 0
+  const isBranchingEnabled = project.preview_branch_refs && project.preview_branch_refs.length > 0
```

Comment:
i'd opt to do it this way tbh

```suggestion
  const isBranchingEnabled = (project?.preview_branch_refs ?? []).length > 0
```

Reply:
Hi @joshenlim - Shall I create a commit for this?